Source/WebGPU/WebGPU/Queue.mm

```diff
@@ -124,7 +124,11 @@
     if (!isValid())
         return std::make_pair(nil, nil);
 
+#if PLATFORM(IOS)
```

Comment:
Why don't we want to raise the limit on iOS?

Reply:
iOS will jetsam around 2000 buffers with no other resource usage. So we will need to do something else, though possibly the CTS gets updated prior to that

Reply:
I'm a bit confused by this too... are we doing this to pass the CTS test, which is incorrect/outdated?

Reply:
Yes that is correct. Application should not create more than 1000 per https://github.com/gpuweb/gpuweb/issues/4622 or risk device lost. CTS is not handling device lost (it should see the device is lost, pause the test, re-create the device, resume the test cases).

Reply:
What happens today is too many tests run at once, CTS never calls GPUCommandEncoder.finish or GPUQueue.submit on the invalid encoders, we retain the MTLCommandBuffer reference until the JS object gets GC'ed from the test.

At 1000 we would currently return nil and report device lost, so those test cases failed.

Reply:
I see, and I assume `Device::maxCommandBufferCount()` is bigger than 1000, which is causing the flakiness? Should `Device::maxCommandBufferCount()` be updated to be 1000?

Reply:
The flakiness occurs because the CTS does not handle device lost. If we set maxCommandBufferCount to 1000 we would still see the issue, as we would trigger a device lost but the CTS does not respond to that.

Reply:
Oh! I get it now, that makes sense, thanks for walking me through it 😅

Reply:
No problem :)

Reply:
I actually should have investigated this more, we reached the MTLCommandBuffer limit because invalid CommandEncoder instances were retaining their MTLCommandBuffer due to incorrect logic in CommandEncoder::finish.

Increasing this limit is therefore not needed, this change can be reverted